Loop/Managers/Alerts/StoredAlert.swift

```diff
@@ -22,7 +22,6 @@ extension StoredAlert {
             managerIdentifier = alert.identifier.managerIdentifier
             triggerType = alert.trigger.storedType
             triggerInterval = alert.trigger.storedInterval
-            isCritical = alert.foregroundContent?.isCritical ?? false || alert.backgroundContent?.isCritical ?? false
```

Comment:
`interruptionLevel` is stored in the `foregroundContent` and `backgroundContent` fields.